core_perception/ndt_cpu/package.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
   <name>ndt_cpu</name>
   <version>1.12.0</version>
   <description>The ndt_cpu package</description>
   <maintainer email="yuki@ertl.jp">Yuki Kitsukawa</maintainer>
   <maintainer email="anh@ertl.jp">Anh Viet Nguyen</maintainer>
   <license>Apache 2</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>  
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake_auto</buildtool_depend>
 
   <build_depend>libpcl-all-dev</build_depend>
 
```

Comment:
Maybe we should also add

```
  <export>
    <build_type condition="$ROS_VERSION == 1">catkin</build_type>
    <build_type condition="$ROS_VERSION == 2">ament_cmake</build_type>
  </export>
```
  
  Just in case